app/MetadataResolver/DLsiteResolver.php

```diff
@@ -99,9 +101,13 @@ public function resolve(string $url): Metadata
         // 作者名単体の場合もあるし、"作者A / 作者B"のようになることもある
         $makersNode = $xpath->query('//*[@id="work_maker"]//*[contains(text(), "' . $makers[0] . '")]/ancestor::td')->item(0);
         $makersArray = [];
-        foreach ($makersNode->getElementsByTagName('a') as $makerNode) {
-            $makersArray[] = trim($makerNode->textContent);
+        foreach ($makersNode->childNodes as $makerNode) {
+            // 何らかのタグ(a, span)の場合のみ処理
+            if ($makerNode->nodeType === 1) {
```

Comment:
[定数](https://www.php.net/manual/ja/dom.constants.php)使いたいです

```suggestion
            if ($makerNode->nodeType === XML_ELEMENT_NODE) {
```

Reply:
単純に定数でできるのを知らなかっただけでした
コミットしました